trajectory_msgs/JointTrajectoryPoint[] points
---
swerve_point_generator/Coefs[] orient_coefs
swerve_point_generator/Coefs[] x_coefs
swerve_point_generator/Coefs[] y_coefs
float64[] end_points

